### PR TITLE
Tl/git notes append with no lf

### PR DESCRIPTION
**Pull request by dyrone**
Thanks for taking the time to contribute to Git! Please be advised that the
Git community does not use github.com for their contributions. Instead, we use
a mailing list (git@vger.kernel.org) for code submissions, code reviews, and
bug reports. Nevertheless, you can use GitGitGadget (https://gitgitgadget.github.io/)
to conveniently send your Pull Requests commits to our mailing list.

Please read the "guidelines for contributing" linked above!

### CODE REVIEW

**Comment by dyrone** [comment]
leibao niubi!

**Comment by dyrone** [comment]
leibao

File: .github/workflows/check-whitespace.yml [hunk 1]
```diff
@@ -9,42 +9,83 @@ on:
   pull_request:
     types: [opened, synchronize]
 
+# Avoid unnecessary builds. Unlike the main CI jobs, these are not
+# ci-configurable (but could be).
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```
Comment (dyrone) [inline .github/workflows/check-whitespace.yml]: leibao niubi

File: .github/workflows/check-whitespace.yml [hunk 2]
```diff
@@ -9,42 +9,83 @@ on:
   pull_request:
     types: [opened, synchronize]
 
+# Avoid unnecessary builds. Unlike the main CI jobs, these are not
```
Comment (dyrone) [inline .github/workflows/check-whitespace.yml]: leibao

**Comment by dyrone** [comment]
> leibao niubi!

testsadsadsa

**Comment by misha913loki** [comment]
邮件已成功接收~

**Pull request closed without merging**